Open Judge System/Web/OJS.Web/Areas/Contests/ViewModels/Participants/ParticipantViewModel.cs

```diff
@@ -12,7 +12,7 @@ public class ParticipantViewModel
 
         public ParticipantViewModel(Participant participant, bool official, bool isAdminOrLecturer)
         {
-            this.Contest = ContestViewModel.FromContest.Compile()(participant.Contest);
+            this.Contest = new ContestViewModel();
```

Comment:
Why this is changed?

Reply:
Since we are using cache, now the data is taken only if it is not available in the cache. This is done outside the model (inside the controller), also there are cases when this.contest may return NullReference exception.